### ISSUE TITLE
DevClusterSetup hangs on "Performing Start-Service on: FabricHostSvc"

### ISSUE THREAD

**Issue by pksimeonov** [Azure/service-fabric-issues#1476]
I had an about 18month old version of Service Fabric running locally which I deleted to install the latest version. However, I am unable to get it running - it always hangs for a few minutes on `Performing Start-Service on: FabricHostSvc . This may take a few minutes...` and I get a notification that an error occurred and to check the log (in the attached zip). I tried the below through powershell hoping to get more details, but again just hangs at the same start-service line.

`C:\Program Files\Microsoft SDKs\Service Fabric\ClusterSetup> .\DevClusterSetup.ps1 -CreateOneNodeCluster`

Below is what I've tried so far to fix this... all hang on the `Performing Start-Service` line and the `FabricHostSvc` service is always stuck `Starting`, even after the notification that the start failed and I should check the log. The other odd bit is that the tray manager says SF is running and I have 1 node setup, but the "Manage local cluster" option is greyed out.

* `Reset Local Cluster` from the SF tray manager.
* `Remove Local Cluster` and again setup a 1 or 5 node SF from the SF tray manager.
* Installing a slightly older version of the runtime and sdk which work fine for a colleague
* Using a 5 node cluster setup instead of 1
* Setting `FabricContainerAppsEnabled` to `false` and trying again to setup local dev cluster.
* Providing `NETWORK SERVICE` write access to `C:\ProgramData\Microsoft\Crypto\RSA\MachineKeys` and trying again to setup local dev cluster.
* Changing the `iPAddress` in `ClusterManifestTemplate.json` to `127.0.0.1` and to `localhost` and trying again to setup local dev cluster.
* Changing the location of the Data and Log folders and trying again to setup local dev cluster.
* All of the above config changes together and trying again to setup local dev cluster.

I just uninstalled Service Fabric (which also removed its Event Log folder), deleted any folders related to SF, installed again the latest version through the web installer (`MicrosoftAzure-ServiceFabric-CoreSDK.exe`) and from the tray manager selected 1 Node mode and waited till it failed, retried by itself, and failed again (ie gave me the notification that something went wrong and to check the log). I didn't do anything special like the above fix attempts. The attached zip contains: 
* the exported event logs from `Event Viewer (Local) -> Applications and Services Logs -> Microsoft-Service Fabric -> Admin`. The `Operational` section (next to `Admin`) was empty.
* The contents of `C:\SFDevCluster\Log` folder.

[ServiceFabricIssue.zip](https://github.com/Azure/service-fabric-issues/files/2984417/ServiceFabricIssue.zip)

I'm on Windows 10 Pro Version 10.0.17134 Build 17134

Edit: At some point during the setup it asked me to give it access in Windows Firewall - I allowed it for both Public and Private connections.

**Comment by anantshankar17** [comment]
From the traces it seems that the docker service is not starting. Can you try resetting the docker to factory settings ?

**Comment by pksimeonov** [comment]
@anantshankar17 Reset to factory default didn't help, same behaviour as before. Uninstalling docker did resolve the issue though. I don't use docker for any of the Service Fabric projects, but I do use it for other stuff, so I still need them both. Is there a way to tell SF to ignore Docker or we need to find why SF won't play nice with docker?

**Comment by anantshankar17** [comment]
Docker works with SF. May be your installation was corrupted so the docker service start wasnt working. Remove the cluster again and reinstall docker.

**Comment by pksimeonov** [comment]
I haven't had any issues with Docker or its services and I've had it for quite a while, haven't updated it recently either, so I'm not convinced the fault lies at docker. Everything was working fine until I uninstalled SF and tried to install the latest version.

In any case, I removed the cluster, removed docker, installed latest docker release, and setup the 1 node cluster again - everything seems to be back to normal. Thank you for your help!
